modules/covid_vaccine/app/models/covid_vaccine/v0/expanded_registration_submission.rb

```diff
@@ -49,9 +51,13 @@ class ExpandedRegistrationSubmission < ApplicationRecord
         reg.form_data&.symbolize_keys!
       end
 
-      attr_encrypted :form_data, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller
-      attr_encrypted :raw_form_data, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller
-      attr_encrypted :eligibility_info, key: Settings.db_encryption_key, marshal: true,
+      attr_encrypted :form_data, key: proc { |r|
+                                        r.encryption_key(:form_data)
+                                      }, marshal: true, marshaler: JsonMarshal::Marshaller
+      attr_encrypted :raw_form_data, key: proc { |r|
+                                            r.encryption_key(:raw_form_data)
+                                          }, marshal: true, marshaler: JsonMarshal::Marshaller
+      attr_encrypted :eligibility_info, key: proc { |r| r.encryption_key(:eligibility_info) }, marshal: true,
```

Comment:
super nitpicky but `r.encryption_key(:eligibility_info)` could drop to the next line like the other examples above it.

Reply:
Ah, yes! I think that one must be JUST under the rubocop line limit!

Reply:
Welp, rubocop didn't like [my change](https://github.com/department-of-veterans-affairs/vets-api/runs/3221980173), reverting